erp_tutorial_hnn_core.ipynb

```diff
@@ -1005,13 +912,15 @@
     "\n",
     "* View the evoked responses for different values of the smoothing parameter; how does this affect the simulation?\n",
     "\n",
-    "* View the average evoked response generated by running more trials in a simulation; how does this affect the simulation?"
+    "* View the average evoked response generated by running more trials in a simulation; how does this affect the simulation?\n",
+    "\n",
+    "* Try importing another network from the [templates provided](https://jonescompneurolab.github.io/hnn-core/stable/api.html#network-models-hnn-core). Can you figure out how these models are different from the original model? Try running a simulation with these models."
```

Comment:
just an idea ... feel free to remove it if you think it will open a can of worms

Reply:
love this actually. I doubt we'll have time to get to this during the actual tutorial, but it's a great way to tie in the different model templates and why they're important.